src/shared/logMessages.ts

```diff
@@ -2,6 +2,9 @@ export enum LogErrors {
   collectMetricsSignatureNotRecognized = "No collect-metrics function found for this event.",
   wrongResolvedEnhancedEvents = "Enhanced Events are invalid.",
   networkErrorCommitsPerPR = "Could not fetch commits for PR-Id: '%s'",
+  networkErrorDeployments = "Could not fetch deployments for owner/repo/env: '%s'",
+  networkErrorPackageJson = "Could not fetch pakcage.json for owner/repo: '%s'",
+  parseErrorPackageJson = "Could not parse package.json content.",
```

Comment:
I think it would make sense to also include the owner/repo also for this log message, so we are able to find out why the file could not be parsed.